### ISSUE TITLE
What is userOperator for?

### ISSUE THREAD

**Issue by areedv** [Rapporteket/ablanor#2]
@skkrist hva er tenkt med denne? Til info, og om det er relevant: ved innlogging til Rapporteket kan det være mange brukere per resh-id og brukernavnene som benyttes har ingen kobling til registerdata. Hvis det her er behov for "personlige rapporter" så må det etableres en slik kobling, eksempelvis i form av interne data i R-pakken

**Comment by skkrist** [comment]
Ideen er at alle operatørene kan få oversikt over sin egen aktivitet. 

Se rad 110 i månedsrapporten : 

RAPPORT PER BRUKER
if(params$userRole == "LU") {
  d_ablanor %<>% dplyr::filter(operator == params$userOperator)
}
RAPPORT PER SYKEHUS
if(params$userRole == "LC"){
  d_ablanor %<>% dplyr::filter(as.numeric(centreid) == params$reshId)
}

Men f.eks om det er er sykepleier som registrer prosedyren er det ikke han som skal ha rapporten, derfor er variabel "operator" brukt. 
Operator er numerisk variabel "PROS_OPERATOR", hvert nummer er en operatør. Ideen er at dersom en av operatørene på listen logger seg på rapporteket med LU rolle, kan han få sin egen aktivitet. Tenker en liste/kobling skal være mulig å få til. Ikke første prioritet!

**Comment by skkrist** [comment]
Menyen må bare vises dersom LU-rolle + en av personene i listen (server.R)

**Comment by areedv** [comment]
Da må det altså lages/vedlikeholdes et eget datasett som kobler brukernavnet (som Rapporteket henter fra innloggingen) med PROS_OPERATOR (som Rapporteket henter fra registerdata), om jeg har forstått det riktig

**Comment by skkrist** [comment]
Ja, høyres ut som en løsning. Skal listen da ligge på Rapporteket? Vedlikehold faller på statistiker?

**Comment by areedv** [comment]
Så lenge brukernavn og PROS_OPERATOR ikke er sensitiv informasjon (i kombinasjon) så kan det enkleste være å legge det  inn som et internt datasett i R-pakke. I så fall må det vedlikeholdes av oss (altså de som håndterer R-pakken). Alternativt kan det legges inn i databasen som en egen tabell sammen med registerdata der det kan vedlikeholdes som en del av den daglige registeradministrasjonen. Det siste er i prinsippet mer riktig da Rapporteket selv kun er en ut-kanal for data (og skal ikke brukes til registrering av data) men det krever da et utviklingsløp mellom registeret og hnikt

**Comment by skkrist** [comment]
OK, vi kan vente litt med denne saken
